gammapy/cube/core.py

```diff
@@ -521,6 +545,33 @@ def reproject(self, reference, mode='interp', *args, **kwargs):
         return self.__class__(name=self.name, data=data, wcs=wcs, meta=self.meta,
                               energy_axis=self.energy_axis)
 
+    def convolve(self, kernels, **kwargs):
+        """
+        Convolve cube with a given energy dependent PSF.
+
+        Parameters
+        ----------
+        kernels : list
+            List of 2D convolution kernels.
```

Comment:
What are kernels? Numpy arrays? Astropy kernel objects?
Above you say:
> Convolve cube with a given energy dependent PSF.
which suggests the input would be such a PSF object, which I guess it isn't?

Can you make this more clear in the docstring or put an example here or in the high-level docs and link to it from here?